float64[10] q
float64[10] tau
float64[10] ee_Ftask
int32 dim_q
float64[16] o_T_ee
float64[6] ee_F_ee
float64 time
